validator/accounts/account_test.go

```diff
@@ -155,3 +176,267 @@ func TestChangePassword_KeyNotMatchingOldPasswordNotEncryptedWithNewPassword(t *
 		t.Error("Key incorrectly encrypted using the new password")
 	}
 }
+
+func TestMerge(t *testing.T) {
```

Comment:
These tests should be in the db , where the actual method is defined.

Reply:
I already spoke with @terencechain about this and we decided on creating duplicate tests in the `db` package. Do you think that testing from the accounts level is unnecessary and I should move the tests from here to db?

Reply:
Seeing as the majority of the logic is in the db package, it doesn't make sense to test them in accounts. You could have maybe a basic sanity test in accounts, but the core logic should be tested in `db` .